tenzir/integration/data/reference/expression/test_universal_function_call_syntax/step_01.ref

```diff
@@ -0,0 +1,8 @@
+error: expected additional positional argument `<string>`
+ --> /dev/stdin:1:17
+  |
+1 | from { x: "abc".starts_with() }
+  |                 ^^^^^^^^^^^ 
+  |
+  = usage: starts_with(<string>, <string>)
+  = docs: https://docs.tenzir.com/functions/starts_with
```

Comment:
I followed to approach of the docs and use function-style usage here, even when it's called as a method. Might be a bit confusing, but otherwise it's inconsistent? Thoughts?

Reply:
I think this is generally a good idea for consistency.

Reply:
So keeping it as-is?

Reply:
Yes.